src/main/java/seedu/apollo/ui/Parser.java

```diff
@@ -136,10 +146,16 @@ private static Command parseCommand(String[] split, int size, ModuleList moduleD
             return new ExitCommand();
 
         case COMMAND_HELP_WORD:
-            if (!isOneWord(split)) {
+            if (!isOneWord(split) && (!isValidWord(split[1]))){
```

Comment:
Why not just combine this with the HelpCommand creation you do later? No need to have two switch cases